package.json

```diff
@@ -165,6 +165,11 @@
     "webpack-cli": "^4.5.0",
     "yorkie": "^2.0.0"
   },
+  "overrides": {
+  	"ajv": {
+      "uri-js": "npm:fast-uri@^2.3.0"
```

Comment:
Is this the correct syntax? Do we need "npm:" here?

Reply:
Yes, the npm is required to tell the package.json that it should retrive a different package